lua/fidget/integration/nvim-tree.lua

```diff
@@ -39,7 +39,7 @@ require("fidget.options").declare(M, "integration.nvim-tree", M.options, functio
       local col = vim.api.nvim_win_get_position(winid)[2]
       if col > 1 then
         local width = vim.api.nvim_win_get_width(winid)
-        win.set_x_offset(width)
+        win.set_x_offset(width + 1)
```

Comment:
`+ 1` was missing, I also fixed it for `nvim-tree`. However, I'm not fully sure why. The calculation is:
```lua
col = math.max(0, col - M.options.x_padding - state.x_offset)
```
So probably something is wrong with the math somewhere.

Reply:
Oh you're so right. I just took a closer look at nvim-tree and I totally mistook the nvim-tree drawer border for a column in the text buffer.